### PR TITLE
[CPU] [TESTS] returned old behavior for fp32 avx2 1x1 conv with dw conv fusing

### PR DESCRIPTION
**Pull request by antonvor**
This PR contains tests for PR https://github.com/openvinotoolkit/openvino/pull/4484

### CODE REVIEW

**Review by maxnick — APPROVED** [review]

**Comment by dmitry-gorokhov** [comment]
@antonvor Is the PR ready for review?

**Comment by antonvor** [comment]
> @antonvor Is the PR ready for review?

File: inference-engine/tests/functional/plugin/cpu/subgraph_tests/src/conv_dw_conv_fusing.cpp [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+const SizeVector inputShape2D{1, 96, 224, 224};
+const size_t numOutChannels = 96;
+
+std::vector<ConvDWConvPatternParams> ConvDWConvPatternParams2DSet = {
+        ConvDWConvPatternParams{numOutChannels, 1, true, true},
+        ConvDWConvPatternParams{numOutChannels, 1, false, true},
+        ConvDWConvPatternParams{numOutChannels, 2, true, true},
+        ConvDWConvPatternParams{numOutChannels, 2, false, true},
+        // todo: [antonvor] dw convolution without biases not supported
```
Comment (dmitry-gorokhov) [inline inference-engine/tests/functional/plugin/cpu/subgraph_tests/src/conv_dw_conv_fusing.cpp:107]: Lets remove than

File: inference-engine/tests/functional/plugin/cpu/subgraph_tests/src/conv_dw_conv_fusing.cpp [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+    std::tie(numOutChannels, dwStride, withBias, withDWBias) = convDWConvPatternParams;
+
+    auto inputParams = ngraph::builder::makeParams(ngraph::element::f32, {inputShape});
+    auto paramOuts = ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(inputParams));
+
+    auto conv1 = ngraph::builder::makeConvolution(paramOuts[0], ngraph::element::f32, {1, 1}, {1, 1}, {0, 0},
+                                                  {0, 0}, {1, 1}, ngraph::op::PadType::EXPLICIT, numOutChannels, withBias);
+
+    auto conv2 = ngraph::builder::makeGroupConvolution(conv1, ngraph::element::f32, {3, 3}, {dwStride, dwStride}, {1, 1}, {1, 1}, {1, 1},
```
Comment (dmitry-gorokhov) [inline inference-engine/tests/functional/plugin/cpu/subgraph_tests/src/conv_dw_conv_fusing.cpp:83]: We also need to cover:
1) Cases with non fp32 prevision.
2) Cases when Conv and DW conv have other fusings: Eltwise/Depthwise/Sum.

**Review by dmitry-gorokhov — CHANGES_REQUESTED** [review]

**Comment by dmitry-gorokhov** [comment]
@antonvor can we close this one as not actual anymore?

**Pull request closed without merging**